tests/suites/test_suite_mdx.function

```diff
@@ -2,6 +2,7 @@
 #include "mbedtls/md2.h"
 #include "mbedtls/md4.h"
 #include "mbedtls/md5.h"
+#include "mbedtls/sm3.h"
```

Comment:
Please add SM3 test vectors to `test_suite_md.data` so as to test the dispatch via the `mbedtls_md_xxx` interface.